src/main/java/ArrayEditing.java

```diff
@@ -0,0 +1,55 @@
+/*
+1.4. Удаление из массива
+
+        Задача:
+        Дан массив целых чисел и ещё одно целое число. Удалите все вхождения этого числа из массива (пропусков быть не должно).
+*/
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class ArrayEditing {
+
+    public static void main (String[] args) {
+        //ArrayList<Integer> createdArrayForEditing =  MaxMinAverage.generateRandomArray(20);
+        int[] createdArrayForEditing = {4, 5, 0, 5, 6, 0, 8, 3, 3, 0, 2, 6, 5, 4, 3, 0, 4, 6, 6, 9};
+        System.out.println("Initial array: " + Arrays.toString(createdArrayForEditing));
+        System.out.println("Updated array: " + Arrays.toString(removeNumberFromArray(createdArrayForEditing,4)));
+    }
+
+
+    //This method removes element that should be removed AND THE NEXT ONE FOR SOME REASON. Investigating it.
+    private static int[] removeNumberFromArray(int[] inputArray, int numberToEdit){
```

Comment:
numberToEdit - is there any other name that better represent the purpose of this variable?

Reply:
variable was renamed to numberToRemove